petastorm/tests/test_common.py

```diff
@@ -50,6 +50,7 @@
     UnischemaField('matrix_nullable', np.uint16, _DEFAULT_IMAGE_SIZE, NdarrayCodec(), True),
     UnischemaField('sensor_name', np.unicode_, (1,), NdarrayCodec(), False),
     UnischemaField('string_array_nullable', np.unicode_, (None,), NdarrayCodec(), True),
+    UnischemaField('integer_nullable', np.int32, (), ScalarCodec(ShortType()), True),
```

Comment:
Any way we can set it as a flar (like nullable=True) in the future API?

Reply:
We can do it today. Chose not to do since it would be too verbose.